website/content/docs/k8s/installation/install.mdx

```diff
@@ -15,58 +15,54 @@ For pure-Kubernetes workloads, this enables Consul to also exist purely
+for Kubernetes, follow the [Getting Started with Consul service
+mesh](https://learn.hashicorp.com/tutorials/consul/service-mesh-deploy?utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial.
 
 ## Helm Chart Installation
 
 The recommended way to run Consul on Kubernetes is via the
 [Helm chart](/docs/k8s/helm). This will install and configure
 all the necessary components to run Consul. The configuration enables you
-to run just a server cluster, just a client cluster, or both. Using the Helm
-chart, you can have a full Consul deployment up and running in minutes.
+to run a server cluster, a client cluster, or both.
 
 Step-by-step tutorials for how to deploy Consul to Kubernetes, please see
 our [Deploy to Kubernetes](https://learn.hashicorp.com/collections/consul/kubernetes-deploy)
-collection. This collection includes configuration caveats for single node deployments.
+collection. This collection includes configuration caveats for single-node deployments.
 
-While the Helm chart exposes dozens of useful configurations and automatically
-sets up complex resources, it **does not automatically operate Consul.**
-You are still responsible for learning how to monitor, backup,
+The Helm chart exposes several useful configurations and automatically
+sets up complex resources, but it **does not automatically operate Consul.**
+You must still become familiar with how to monitor, backup,
 upgrade, etc. the Consul cluster.
 
 The Helm chart has no required configuration and will install a Consul
-cluster with reasonable defaults out of the box. Prior to going to production,
-it is highly recommended that you
-[learn about the configuration options](/docs/k8s/helm#configuration-values).
+cluster with default configurations. We strongly recommend [learning about the configuration options](/docs/k8s/helm#configuration-values) prior to going to production.
 
 ~> **Security Warning:** By default, the chart will install an insecure configuration
 of Consul. This provides a less complicated out-of-box experience for new users,
-but is not appropriate for a production setup. It is highly recommended to use
-a properly secured Kubernetes cluster or make sure that you understand and enable
+but is not appropriate for a production setup. We strongly recommend using
+a properly-secured Kubernetes cluster or making sure that you understand and enable
 the [recommended security features](/docs/internals/security). Currently,
 some of these features are not supported in the Helm chart and require additional
 manual configuration.
 
 ### Prerequisites
 
-The Consul Helm chart works only with Helm 3. Install the latest version of the Helm CLI here:
+The Consul Helm only supports Helm 3. Install the latest version of the Helm CLI here:
 [Installing Helm](https://helm.sh/docs/intro/install/).
 
 ### Installing Consul
 
-Add the HashiCorp Helm Repository:
+1. Add the HashiCorp Helm Repository:
```

Comment:
The indentation is a bit off here @trujillo-adam 

![image](https://user-images.githubusercontent.com/29551334/133498535-57df14fb-f991-47df-a060-4f7cb89d7354.png)